docs/enhancements/4-nodes-cluster-deployment.md

```diff
@@ -0,0 +1,154 @@
+---
+title: 4-nodes-cluster-creation
+authors:
+  - "@flaper87"
+creation-date: 2022-04-14
+last-updated: yyyy-mm-dd
+---
+
+# Allow for a 4-nodes cluster to be created
+
+# Summary
+
+**Note:** Throughout this enhancement, a 4-nodes cluster refers to a 3 schedulable masters and 1 worker.
+
+There are cases where customers may just have 4 nodes available and no chance to add more. This was
+one of the cases in the ZTP for Factory Workflow where it was required to deploy a cluster in an
+enclosure that only had 4 nodes. The goal was to use all of the blades rather than just 3 of them.
+
+For now, this is achieved with a 2 step deployment, first 3 masters and then add a worker node. It
+would be more time efficient as well as easier and more reliable to be able to do this deployment in
+one step, day0.
+
+# Motivation
+
+Assisted service has built-in validations that prevent the deployment of a 4-nodes cluster. This is
+to prevent unhealthy clusters to be deployed. A 4-nodes cluster ends up in an unhealthy state
```

Comment:
Calling them "unhealthy" challenges my understanding - I expect the clusters to be completely healthy and finish installation successfully. 

They just have single replicas infrastructure topology, which has pros and cons, but I wouldn't call them unhealthy.

Is this correct, or am I missing something?

If I'm correct, then it means the current assisted-service validations for preventing 4-node clusters are not justified and should be removed, regardless of this enhancement